Sources/Web3Modal/Environment+ProjectId.swift

```diff
@@ -0,0 +1,12 @@
+import SwiftUI
+
+private struct ProjectIdKey: EnvironmentKey {
```

Comment:
looks nice, where it's using?

Reply:
Yeah I thought about using it to set projectID but didn't end up using it